packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -68,4 +72,24 @@ export class BlobManager {
 
         return handle;
     }
+
+    public addBlobId(blobId: string) {
+        this.blobs.add(blobId);
+    }
+
+    public load(blobsBlob?: string) {
```

Comment:
can you please put a comment describing what this function is supposed to do / argument description?